synapse/http/client.py

```diff
@@ -782,7 +789,7 @@ def connectionLost(self, reason: Failure) -> None:
             self.deferred.errback(reason)
 
 
-def readBodyToFile(
+def readBodyWithMaxSize(
```

Comment:
is there a reason for the unpythonic camelCase name here? we should probably fix that while we're here.

Reply:
I think it is to match the method from Twisted call `readBody`. I can update it though!